src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -387,32 +387,32 @@ public Job call() throws SaltStackException {
         return executor.submit(callable);
     }
 
-  /**
-  * Query statistics from the CherryPy Server.
-  *
-  * GET /stats
-  *
-  * @return The {@link Stats} object.
-  * @throws SaltStackException if anything goes wrong
-  */
-  public Stats stats() throws SaltStackException {
-      return connectionFactory.create("/stats", JsonParser.STATS, config).getResult();
-  }
-
-  /**
-  * Asynchronously query statistics from the CherryPy Server.
-  *
-  * GET /stats
-  *
-  * @return Future containing the {@link Stats} object.
-  */
-  public Future<Stats> statsAsync() {
-      Callable<Stats> callable = new Callable<Stats>() {
-          @Override
-          public Stats call() throws SaltStackException {
-              return stats();
-          }
-      };
-      return executor.submit(callable);
-  }
+    /**
+     * Query statistics from the CherryPy Server.
+     * <p/>
+     * GET /stats
+     *
+     * @return The {@link Stats} object.
+     * @throws SaltStackException if anything goes wrong
+     */
+    public Stats stats() throws SaltStackException {
+        return connectionFactory.create("/stats", JsonParser.STATS, config).getResult();
+    }
+
+    /**
+     * Asynchronously query statistics from the CherryPy Server.
+     * <p/>
```

Comment:
Hmm, probably the IDE put it there, I don't remember typing it there manually. Will fix in next push.